main/src/hadd.cxx

```diff
@@ -289,6 +289,7 @@ int main( int argc, char **argv )
                }
             }
          }
+         ++ffirst;
```

Comment:
In what circumstance does this bug manifest?

Reply:
If you use the (unreleased) `-experimental-io-features` flag that accidentally snuck into #1003.

Reply:
The commit should really mention this :).  It sounds worth otherwise :)